src/s1reader/s1_burst_slc.py

```diff
@@ -622,3 +627,9 @@ def swath_name(self):
         '''Swath name in iw1, iw2, iw3.'''
         return self.burst_id.split('_')[1]
 
+    @property
+    def bbox(self):
+        '''Returns the (west, south, east, north) bounding box of the burst.'''
+        # Uses https://shapely.readthedocs.io/en/stable/manual.html#object.bounds
+        # Returns a tuple of 4 floats representing (west, south, east, north) in degrees.
+        return self.border[0].bounds
```

Comment:
Can you remind me why we need `border[0]` and not simply` border`?

Reply:
I was wondering that myself too, and was thinking of raising that as an issue:

The comment in `Sentinel1BurstSlc` attribute list says
```
    border: list # list of lon, lat coordinate tuples (in degrees) representing burst border
```
which is no longer true (I assume it was at the beginning)- it's now a list of a single shapely polygon

My only guess is because of this `check_dateline` function: https://github-fn.jpl.nasa.gov/isce-3/isce/blob/f21fa0ff13987a7b71c250e2eead89d0d4dc1ce2/python/packages/nisar/workflows/stage_dem.py#L44  which is being used to set the border https://github.com/opera-adt/s1-reader/blob/main/src/s1reader/s1_reader.py#L260 

Maybe the decision was to use a list of Polygons rather than a MultiPolygon?

I suppose if we keep that as is, the least we should do is update the comment saying it's a list of lat/lon tuples